extra/release_tool.py

```diff
@@ -1859,13 +1859,34 @@ def tag_or_followed_branch(repo_git):
                 query_execute_list(
                     [
                         [
+                            "DOCKER_BUILDKIT=1",
+                            "docker",
+                            "build",
+                            "-t",
+                            "mendersoftware/gui:base",
+                            "-f",
+                            os.path.join(tmpdir, "Dockerfile"),
+                            "--target",
+                            "base",
+                            tmpdir,
+                        ],
+                        [
+                            "DOCKER_BUILDKIT=1",
                             "docker",
                             "build",
                             "-t",
                             gui_tag,
                             "-f",
-                            os.path.join(tmpdir, "Dockerfile.disclaimer"),
+                            os.path.join(tmpdir, "Dockerfile"),
+                            "--target",
+                            "disclaim",
                             tmpdir,
+                        ],
+                        [
+                            "docker",
+                            "image",
+                            "rm",
+                            "mendersoftware/gui:base"
                         ]
```

Comment:
Maybe keeping this image around is not a bad idea. During the release process we would run this command at least two times, so it would be nice to reuse the cached image the second time if there are no changes in the gui repo.

If you decide to remove it to have the system clean, okey, but then let's remove also `gui_tag` once we are done with the `docker run` command.

Reply:
In this case I'll keep it for the subsequent runs 🏃